packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1ArbitrumGateway.sol

```diff
@@ -217,8 +217,9 @@ abstract contract L1ArbitrumGateway is L1ArbitrumMessenger, TokenGateway {
     /**
      * @notice Deposit ERC20 token from Ethereum into Arbitrum. If L2 side hasn't been deployed yet, includes name/symbol/decimals data for initial L2 deploy. Initiate by GatewayRouter.
      * @param _l1Token L1 address of ERC20
-     * @param _refundTo account to be credited with the excess gas refund in the L2, subject to L2 alias rewrite if its a L1 contract
-     * @param _to account to be credited with the tokens in the L2 (can be the user's L2 account or a contract)
+     * @param _refundTo Account, or its L2 alias if it have code in L1, to be credited with excess gas refund in L2
```

Comment:
we could also add a heads-up about the edge-cases of the code showing up in L1 (ie no code when deployin, and still code after self destruct)